src/routes/Apps/App/Testing.tsx

```diff
@@ -315,8 +295,50 @@ class Testing extends React.Component<Props, ComponentState> {
     async startTest(sourceName: string): Promise<void> {
         if (this.state.testSet) {
             const testItems = this.state.testSet.getTestItems(sourceName)
-            await Util.setStateAsync(this, { testItems, testIndex: 0 })
-            await this.testNextTranscript()
+            await Util.setStateAsync(this, { testItems, completedTestIds: [], doneCount: 0 })
+            const testSlots: string[] = []
+            for (let i = 0; i < NUM_PARALLEL_TESTS; i = i + 1) {
+                testSlots.push(`ValidationTest ${i}`)
+            }
+            await Util.setStateAsync(this, {testSlots})
```

Comment:
nit: formatting

Reply:
what formatting change are you suggesting?

Reply:
Did you try formatting it and seeing what changed? It's built into VSCode.
We do this so these changes don't keep showing up on other peoples reviews and distracting from the actual content of review.

You need spaces around curly braces
`await Util.setStateAsync(this, { testSlots })`

Reply:
I will look at enabling `prettier` again so we can avoid these conversations. They happening way too often. Last time we tried it seemed to change the code too much so it was turned off but saving the team from all this time would be more important.